scripts/zones/Beaucedine_Glacier/npcs/Akane_IM.lua

```diff
@@ -15,10 +17,12 @@ local guardEvent  = 32761
 
 entity.onTrade = function(player, npc, trade)
     xi.conq.overseerOnTrade(player, npc, trade, guardNation, guardType)
+    xi.garrison.onTrade(player, npc, trade, guardNation)
 end
 
 entity.onTrigger = function(player, npc)
     xi.conq.overseerOnTrigger(player, npc, guardNation, guardType, guardEvent, guardRegion)
+    xi.garrison.onTrigger(player, npc)
 end
```

Comment:
These sort of things should probably not run immediately after each other, even if the implementations inside these functions are safe to do so.

```lua
if xi.conq.overseerOnTrade(player, npc, trade, guardNation, guardType) then
    return
else if xi.garrison.onTrade(player, npc, trade, guardNation) then
    return
end
```

This guarantees that only one of these runs, never both. Looks like all the guards need this

Reply:
so the trade items dont overlap I found no issues getting both to work but I will double check this

Reply:
It doesn't matter if they logically won't overlap, at a glance it's impossible to tell if you're a passer-by. These should be split in a way that makes it obvious that only one can be firing at a time (if/else blocks)